lib/cog/models/chat_handle.ex

```diff
@@ -19,8 +19,16 @@ defmodule Cog.Models.ChatHandle do
   detail_fields [:id, :handle, :user, :chat_provider, :chat_provider_user_id]
 
   def changeset(model, params \\ :empty) do
+    params = coerce_chat_provider_user_id(params)
+
     model
     |> cast(params, @required_fields, @optional_fields)
   end
 
+  defp coerce_chat_provider_user_id(%{"chat_provider_user_id" => chat_provider_user_id} = params),
```

Comment:
I was initially thinking something along the lines of https://github.com/operable/cog/blob/51546b25be6e503913e8994d0afff3243336072c/lib/cog/models/user.ex#L81-L99 and putting that into the pipeline for the changeset. That way you'd get an error message if it failed.